pkg/operator/vspherecontroller/vspherecontroller_test.go

```diff
@@ -31,7 +30,7 @@ const (
 )
 
 func newVsphereController(apiClients *utils.APIClient) *VSphereController {
-	gates := featuregates.NewFeatureGate([]configv1.FeatureGateName{"SomeEnabledFeatureGate"}, []configv1.FeatureGateName{"SomeDisabledFeatureGate", features.FeatureGateVSphereMultiVCenters})
+	gates := featuregates.NewFeatureGate([]configv1.FeatureGateName{"SomeEnabledFeatureGate"}, []configv1.FeatureGateName{"SomeDisabledFeatureGate"})
 	return newVsphereControllerWithGates(apiClients, gates)
```

Comment:
If we are not using the featuregate checks in the test, we should simply remove the field.

Reply:
Had to add this back (slightly modified) due to new feature gate that was added (FeatureGateVSphereConfigurableMaxAllowedBlockVolumesPerNode).  This needs to be initialized for tests to run w/o panic.